tests/system/action/assignment/test_update.py

```diff
@@ -111,6 +111,20 @@ def prepare_voting_phase_test(
         )
 
     def test_update_phase_to_voting(self) -> None:
+        self.prepare_voting_phase_test(3)
+        self.request("speaker.create", {"meeting_user_id": 1, "list_of_speakers_id": 1})
+        response = self.request(
+            "assignment.update",
+            {"id": 1, "phase": "voting"},
+        )
+        self.assert_status_code(response, 200)
+        for id_ in [1, 2, 3]:
+            self.assert_model_exists(
+                f"speaker/{id_}",
+                {"list_of_speakers_id": 1, "meeting_user_id": id_, "meeting_id": 1},
+            )
```

Comment:
Mmh, I think you want to check the los here too. That it just have three speakers in it. Also check the los in the test below.